.github/workflows/require-pr-label.yml

```
@@ -0,0 +1,16 @@
+name: Require PR label
+
+on:
+  pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: mheap/github-action-required-labels@v3
```

I was briefly concerned that we're pulling a non-official action and not [securing the reference of a third-party action](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions).

We're already exposed though, due to `hendrikmuhs/ccache-action@v1.2`.

---

This can use a commit hash to minimise the risk.

---

FYI, v4.0.0 was released recently, so we might want to upgrade:

```suggestion
      - uses: mheap/github-action-required-labels@v4
```

If we want to pint to an exact commit hash:

```suggestion
      - uses: mheap/github-action-required-labels@e33092117bdd8c435e9b654d53726c6936f19518
```